poetry.lock

```diff
@@ -252,7 +252,7 @@ graph = ["objgraph (>=1.7.2)"]
 
 [[package]]
 name = "eq-translations"
-version = "4.9.0"
+version = "4.8.0"
```

Comment:
The packages are already up-to-date, just your local one wasn't. You shouldn't revert the lock file and run `poetry update` locally. It has actually reverted it.

Reply:
Oh hahaha. Okay its back to normal now 😌